python/activator/activator.py

```diff
@@ -347,9 +347,9 @@ def next_visit_handler() -> tuple[str, int]:
     try:
         try:
             expected_visit = parse_next_visit(flask.request)
-        except ValueError as msg:
-            _log.exception()
-            return f"Bad Request: {msg}", 400
+        except ValueError as e:
+            _log.exception("Bad Request: %s", e)
```

Comment:
Sorry, but this seems incorrect to me -- `Logger.exception` normally does not need to take the exception object, because [that's pulled automatically from the handling context](https://docs.python.org/3/library/logging.html#logging.Logger.exception). Doesn't this lead to duplicate exception messages?

Reply:
But it does require an argument. Or do you mean removing this line here and just letting error handler handle later?

Reply:
No, he's saying:

```python
_log.exception("Bad request")
```
instead.

For example if you include the exception string:

```
WARNING:test:Message one
ERROR:test:Bad code: Message 2
Traceback (most recent call last):
  File "/Users/timj/work/lsstsw/build/daf_butler/y.py", line 9, in <module>
    raise RuntimeError("Message 2")
RuntimeError: Message 2
Finish
```
so you get "Message 2" in the first line of the log output and the final line of the output.

Reply:
I see. Thank you for explaining.  
New PR: https://github.com/lsst-dm/prompt_processing/pull/214 
Would one of you review please?